stellarphot/core.py

```diff
@@ -1,14 +1,22 @@
+import re
+
 from astropy import units as u
 from astropy.coordinates import EarthLocation, SkyCoord
 from astropy.table import Column, QTable, Table
 from astropy.time import Time
 from astropy.units import Quantity
+from astropy.wcs import WCS
+
+from astroquery.vizier import Vizier
 
+import pandas as pd
 from pydantic import BaseModel, root_validator
 
 import numpy as np
 
 __all__ = ['Camera', 'BaseEnhancedTable', 'PhotometryData', 'CatalogData',
+           'apass_dr9',
+           'vsx_vizier',
```

Comment:
I put it in `core` since they are basically class factories for `CatalogData`.

I'm currently thinking `core` should be split into separate files. Perhaps `core` retains `BaseEnhancedTable` and `PhotometryTable` and the other two classes split off into separate files. Haven't really thought it through.

It would definitely be nice to group the tests better either by splitting them into separate files or using classes in the single file. It would be great if you could see at a glance which class has a problem.

I haven't thought any of this through really, will try to come up with a concrete proposal.